stubs/CMakeLists.txt

```diff
@@ -14,6 +14,16 @@ ENDIF()
 
 PROJECT(@lib_name@)
 
+# check for and enable c++11 support
+INCLUDE(CheckCXXCompilerFlag)
+CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+IF(COMPILER_SUPPORTS_CXX11)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Yes, I know that you may use the switch that you specify here on Mac.  That isn't what I was saying though. 

My claim is that when everything else that you try to link against uses `-stdlib=libc++` but an object was compiled with `-std=c++11`, the linking will fail. The ABIs are actually different for these two and they are not interchangeable. 

Conda-forge has decided that `-stdlib=libc++` is the standard (see the toolchain link) across all packages. Going against this will likely break the conda package.  If you would like to show that it doesn't, please open up a PR against conda-forge/cyclus-feedstock that pulls in this changeset to show that it doesn't.  Otherwise, I feel it is far too risky to merge this change.